engine/class_modules/monk/sc_monk.cpp

```diff
@@ -3631,34 +3696,71 @@ struct breath_of_fire_t : public monk_spell_t
   double action_multiplier() const override
   {
     double am = monk_spell_t::action_multiplier();
+
+    if ( blackout_combo )
+      am *= 1.0 + p()->buff.blackout_combo->data().effectN( 5 ).percent();
+
```

Comment:
Blackout Combo is already handled via `parse_effect` here:
https://github.com/simulationcraft/simc/blob/560c527933beab95f893c047c63f489a04377635/engine/class_modules/monk/sc_monk.cpp#L169

Reply:
Totally missed that, but I don't believe this is working. Given a simple APL:

```
actions=auto_attack
actions+=/blackout_kick
actions+=/breath_of_fire
```

Breath of Fire's damage is the same even if I switch Blackout Combo (`CwQAAAAAAAAAAAAAAAAAAAAAAAAAAAjxYGGWmx2MzMDAAAAAAALLgYmBmhBGjtZmZYWGzM2mlZabb2sNzMMLAAwGCAAgZbWamZmFmhFA`) for Pretense of Instability (`CwQAAAAAAAAAAAAAAAAAAAAAAAAAAAjxYGGzyM2mZmZAAAAAAAYZBEzMwMMwYsNzMDzyYmx2sMTbbzmtxMMLAAwGCAAgZbWamZmFmhFA`)

Reply:
You'll also notice the same issue with Dragonfire Brew prior to 24aed20eb0122de1157c1f1bd14164bc3af61fc5.

Moving Blackout Combo `expire` prior to `execute` causes the `parse_effect` buff check to always fail.

Shifting it to after should not be a problem given that `impact` does not reference the buff state, only the locally held bool.